.github/workflows/contracts-e2e-tests-and-deploy.yaml

```diff
@@ -92,22 +90,15 @@ jobs:
         env:
           AWS_REGION: us-east-1
 
-      - name: Run one-node smartnet
+      - name: Run nodes
         shell: bash
         run: |
-          .github/scripts/run_smartnet.sh &
+          scripts/run_nodes.sh
```

Comment:
I've found that `run_nodes.sh -v 1 -n 1 -b false` works as well but it runs minimal network of 2 nodes (with 1 validator).

Reply:
Well, this doesn't work on CI, so I'll have to figure something else out tomorrow... I'm not really working today, just listening to meetings :D

Reply:
The main difference being that `run_smartnet.sh` runs a docker network while `run_nodes.sh` plain binaries.

Reply:
This is moot now, I rolled the contract tests into the main `e2e` workflow.